src/tkn.ts

```diff
@@ -42,662 +47,14 @@ function createConfig(): humanize.HumanizerOptions {
-      switch (this.state) {
-        case 'False': {
-          filePath = ERROR_PATH;
-          break;
-        }
-        case 'True': {
-          filePath = IMAGES;
-          break;
-        }
-        default: {
-          return Uri.file(path.join(__dirname, IMAGES, 'running.gif'));
-        }
-      }
-      return Uri.file(path.join(__dirname, filePath, this.CONTEXT_DATA[this.contextValue].icon));
-    }
-    return Uri.file(path.join(__dirname, IMAGES, this.CONTEXT_DATA[this.contextValue].icon));
-  }
-
-  get tooltip(): string {
-    return format(this.CONTEXT_DATA[this.contextValue].tooltip, this);
-  }
-
-  get label(): string {
-    return this.name;
-  }
-
-  getName(): string {
-    return this.name;
-  }
-
-  getChildren(): ProviderResult<TektonNode[]> {
-    return this.CONTEXT_DATA[this.contextValue].getChildren();
-  }
-
-  getParent(): TektonNode {
-    return this.parent;
-  }
-
-  refresh(): Promise<void> {
-    return Promise.resolve();
-  }
-
-}
-
-export type PipelineTaskRunData = {
-  metadata?: {
-    creationTimestamp: string;
-    name: string;
-    uid: string;
-    labels: {
-      'tekton.dev/pipelineTask': string;
-      'tekton.dev/pipelineRun': string;
-      'tekton.dev/task': string;
-      'tekton.dev/conditionCheck'?: string;
-    };
-  };
-  status?: {
-    completionTime: string;
-    conditions: [{
-      status: string;
-    }];
-  };
-  spec: {
-    taskRef: {
-      name: string;
-      kind: string;
-    };
-  };
-};
-
-
-export class TaskRun extends TektonNodeImpl {
-  private started: string;
-  private finished: string;
-  constructor(parent: TektonNode,
-    name: string,
-    tkn: Tkn,
-    item: PipelineTaskRunData) {
-    super(parent, name, ContextType.TASKRUN, tkn, TreeItemCollapsibleState.None, item.metadata?.uid, item.metadata.creationTimestamp, item.status ? item.status.conditions[0].status : '');
-    this.started = item.metadata.creationTimestamp;
-    this.finished = item.status?.completionTime;
-  }
-
-  get label(): string {
-    return this.name;
-  }
-
-  get description(): string {
-    let r = '';
-    if (this.getParent() && this.getParent().contextValue === ContextType.TASK) {
-      if (this.finished) {
-        r = 'started ' + humanizer(Date.now() - Date.parse(this.started)) + ' ago, finished in ' + humanizer(Date.parse(this.finished) - Date.parse(this.started));
-      } else {
-        r = 'started ' + humanizer(Date.now() - Date.parse(this.started)) + ' ago, running for ' + humanizer(Date.now() - Date.parse(this.started));
-      }
-    } else {
-      if (this.finished) {
-        r = 'finished in ' + humanizer(Date.parse(this.finished) - Date.parse(this.started));
-      } else {
-        r = 'running for ' + humanizer(Date.now() - Date.parse(this.started));
-      }
-    }
-    return r;
-  }
-}
+export const IMAGES = '../../../images';
+export const ERROR_PATH = '../../../images/generated/error';
+export const PENDING_PATH = '../../../images/generated/pending';
```

Comment:
I think it would be better to move this constant to other file, `context-type.ts` for example